### PR TITLE
KBC-3027 Add SynapseDriver with wlm context set option in driver options

### PR DESCRIPTION
**Pull request by zajca**
JIRA: KBC-3027

new `SynapseDriver` which is copy of Doctrine\DBAL\Driver\PDO\SQLSrv\Driver with support for setting wlm_context for each connection

### CODE REVIEW

File: phpstan-baseline.neon [hunk 1]
```diff
@@ -1,5 +1,10 @@
 parameters:
 	ignoreErrors:
+		-
+			message: "#^Parameter \\#3 \\$subject of function str_replace expects array\\|string, string\\|null given\\.$#"
```
Comment (jirkasemmler) [inline phpstan-baseline.neon:4]: nejde to fixnout?

File: tests/Functional/Synapse/Connection/WlmContextTest.php [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+use Doctrine\DBAL\DriverManager;
+use Keboola\TableBackendUtils\Connection\Synapse\SynapseDriver;
+use Tests\Keboola\TableBackendUtils\Functional\Synapse\SynapseBaseCase;
+
+class WlmContextTest extends SynapseBaseCase
+{
+    public function testSynapseWlmContext(): void
+    {
+        $conn = DriverManager::getConnection([
```
Comment (jirkasemmler) [inline tests/Functional/Synapse/Connection/WlmContextTest.php:15]: toto je jen tady pro tento test. Ale nechceme to takto (i za cenu nevyplneneho wlm) pouzit i jinde? At maximum veci bezi pres tento driver? je to sice v SynapseBaseCase, ale v BaseAuthTestCase to neni

**Review by jirkasemmler — CHANGES_REQUESTED** [review]

File: phpstan-baseline.neon (hunk 1 above)
Comment (zajca) [reply phpstan-baseline.neon:4]: asi jo, pravděpodobně setinkový update phpstanu, moc sa mě do toho nechce hrabat

File: tests/Functional/Synapse/Connection/WlmContextTest.php (hunk 2 above)
Comment (zajca) [reply tests/Functional/Synapse/Connection/WlmContextTest.php:21]: Přidal sem to všude + sem přidal testSynapseWlmContextNotSet https://github.com/keboola/php-table-backend-utils/pull/81/commits/ee5cf34ddcbe464c6185770ce2444ca2bf149fa2

File: phpstan-baseline.neon (hunk 1 above)
Comment (zajca) [reply phpstan-baseline.neon]: fixnul sem https://github.com/keboola/php-table-backend-utils/pull/81/commits/57370c600d36de5408a95779a8a89729b6a4e92f

**Review by jirkasemmler — APPROVED** [review]
kod cajk, uvidime co build

**Pull request merged**